tests/test_utils.py

```diff
@@ -122,3 +123,11 @@ def test_invert_nodes_without_prefix(self):
         """Test invert nodes."""
         inverted_df = invert_mappings(df=self.msdf2.df, merge_inverted=False)
         self.assertEqual(len(inverted_df), len(self.msdf2.df.drop_duplicates()))
+
+    def test_inject_metadata_into_df(self):
+        """Test injecting metadata into DataFrame is as expected."""
+        expected_creators = "orcid:0000-0001-5839-2535|orcid:0000-0001-5839-2532"
+        msdf = parse_sssom_table(f"{data_dir}/test_inject_metadata_msdf.tsv")
+        msdf_with_meta = inject_metadata_into_df(msdf)
+        creator_ids = msdf_with_meta.df["creator_id"].drop_duplicates().values.item()
+        self.assertEqual(creator_ids, expected_creators)
```

Comment:
A better test would be to export the df to SSSOM json and check that the field is a list of two orcids. This May be excessive but I just want to be sure the field is formatted correctly. I leave it to your judgement

Reply:
Done